### PR TITLE
deconstruct props/mark up standard fixes

### PR DESCRIPTION
**Pull request by fraser-uno**
Couple small fixes, going by Garry's comments in the Slack channel

### CODE REVIEW

File: web-app/src/components/UserDetails/index.jsx [hunk 1]
```diff
@@ -6,13 +6,14 @@ import { Card } from '../styled'
...
 
+  const { user: { forename, surname, employeeRoleDescription } } = props;
   
   return (
     <Card>
         <strong>User Details</strong>
 
-        <div>Name: { props.user.surname + ', ' + props.user.forename }</div>
-        <div>Role: { props.user.employeeRoleDescription } </div>
+        <div><span>Name: { surname + ', ' + forename }</span></div>
```
Comment (ghost) [inline web-app/src/components/UserDetails/index.jsx]: A little more ES6 syntax needed just `<div><span>Name: { `${surname}, ${forename}` }</span></div>`

You can use that for the others

File: web-app/src/components/Leave/index.jsx [hunk 2]
```diff
@@ -6,12 +6,14 @@ import { Card } from '../styled'
 
 export const Leave = (props) => {
 
+  const {totalHolidays, takenHolidays} = props
```
Comment (ghost) [inline web-app/src/components/Leave/index.jsx:9]: Looks good!

**Review by ghost — APPROVED** [review]
Nicely done Sir.

**Comment by BrownMark** [comment]
Looks good

**Pull request merged**